src/web/upload_task.py

```
@@ -130,110 +109,96 @@ def upload_task(
     main_title: str,
     do_upload: Optional[bool] = None,
     user: Dict[str, str] = None,
-    progress_updater: ProgressUpdater = None,
+    store: TaskStorePyMysql =None,
```

Flake8 (7.3.0)

[error] missing whitespace around parameter equals

(E252)